test/promesa/tests/core_test.cljc

```diff
@@ -710,7 +710,7 @@
              (if (= a 5)
                a
                (p/recur (p/delay 50 (inc a)))))
-        p2 (p/race [p1 (p/delay 400 10)])]
+        p2 (p/race [p1 (p/delay 1000 10)])]
```

Comment:
@niwinz I needed to bump this timeout a bit because on macOS + rosetta2 I did get a flaky test failure with this one